packages/gestalt/src/Video/Video.js

```diff
@@ -0,0 +1,273 @@
+    }
+  };
+
+  // Play/pause the video
+  togglePlay = () => {
+    if (this.video) {
+      if (this.video.paused) {
+        this.video.play();
+      } else {
+        this.video.pause();
+      }
+    }
+  };
+
+  /**
+   * Handlers for various media events on the video
+   */
+
+  // The metadata has loaded or changed, indicating a change in
+  // duration of the media
+  handleDurationChange = (event: SyntheticEvent<HTMLVideoElement>) => {
+    const { onDurationChange } = this.props;
+    this.setState({ duration: (this.video && this.video.duration) || 0 });
+
+    if (onDurationChange) {
+      onDurationChange({ event });
+    }
+  };
+
+  // Sent when the video is switched to/out-of fullscreen mode
+  handleFullScreenChange = (event: Event) => {
+    const { onFullScreenChange } = this.props;
+    this.setState({ isFullscreen: !!fullscreen.isFullscreen() });
+
+    if (onFullScreenChange) {
+      onFullScreenChange({ event });
+    }
+  };
+
+  // Sent when the media begins to play (either for the first time,
+  // after having been paused, or after ending and then restarting).
+  handlePlay = (event: SyntheticEvent<HTMLVideoElement>) => {
+    const { onPlay } = this.props;
+    this.setState({ paused: false });
+
+    if (onPlay) {
+      onPlay({ event });
+    }
+  };
+
+  // Sent when playback is paused.
+  handlePause = (event: SyntheticEvent<HTMLVideoElement>) => {
+    const { onPause } = this.props;
+    this.setState({ paused: true });
+
+    if (onPause) {
+      onPause({ event });
+    }
+  };
+
+  // The time indicated by the element's currentTime attribute has changed
+  handleTimeUpdate = (event: SyntheticEvent<HTMLVideoElement>) => {
+    const { onTimeUpdate } = this.props;
+    this.setState({ currentTime: (this.video && this.video.currentTime) || 0 });
+
+    if (onTimeUpdate) {
+      onTimeUpdate({ event });
+    }
+  };
+
+  // Sent when the audio volume changes
+  handleVolumeChange = (event: SyntheticEvent<HTMLVideoElement>) => {
+    const { onVolumeChange } = this.props;
+    this.setState({ muted: (this.video && this.video.muted) || false });
+
+    if (onVolumeChange) {
+      onVolumeChange({ event });
+    }
+  };
+
+  render() {
+    const { autoPlay, controls, loop, poster, preload, src } = this.props;
+    const { currentTime, duration, isFullscreen, muted, paused } = this.state;
+
+    const paddingBottom =
+      // In full screen the padding bottom is 0 to fit the screen
+      (isFullscreen && '0') ||
+      // If video data is present, use the correct aspect ratio
+      (this.video &&
+        `${this.video.videoHeight / this.video.videoWidth * 100}%`) ||
+      // If the video metadata is missing, default to a standard 16:9 ratio
+      `${9 / 16 * 100}%`;
+
+    return (
+      <div
+        ref={this.setPlayerRef}
+        className={styles.player}
+        style={{ paddingBottom, height: isFullscreen ? '100%' : 0 }}
+      >
+        {/* eslint-disable jsx-a11y/media-has-caption */}
```

Comment:
@pfarejowicz Could we build in support for this instead of disabling the a11y linter?
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/media-has-caption.md